tests/acceptance/expected-failures-API-on-OCIS-storage.md

```diff
@@ -212,14 +212,6 @@ cannot share a folder with create permission
 #### [Public link enforce permissions](https://github.com/owncloud/ocis/issues/1269)
 - [coreApiSharePublicLink1/createPublicLinkShare.feature:327](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/coreApiSharePublicLink1/createPublicLinkShare.feature#L327)
 
-#### [download previews of other users file](https://github.com/owncloud/ocis/issues/2071)
-
-- [coreApiWebdavPreviews/previews.feature:95](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/coreApiWebdavPreviews/previews.feature#L95)
-
-#### [different error message detail for previews of folder](https://github.com/owncloud/ocis/issues/2064)
-
-- [coreApiWebdavPreviews/previews.feature:104](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/coreApiWebdavPreviews/previews.feature#L104)
-
```

Comment:
totally forgot that the fix was in master. but IMO, we have to put tests here for stable